client/src/components/selects/TSelect.tsx

```diff
@@ -61,10 +66,12 @@ export default class TSelect<T> extends React.Component<ISelectProps<T>, any> {
       >
         <Button
           disabled={disabled}
+          fill={true}
+          alignText={'left'}
```

Comment:
Does it change the display somewhere else?
Maybe we could also pass these as optional props to TSelect

Reply:
It does change the display elswhere but it's actualy nicer so I'll keep it that way until we need to make this configirable, how does that sound ?

Reply:
If it makes things nicer, I'm good with that 👍